src/mrpro/data/traj_calculators/KTrajectoryPulseq.py

```diff
@@ -48,19 +56,20 @@ def __call__(self, kheader: KHeader) -> KTrajectoryRawShape:
         k_traj_adc_numpy, _, _, _, _ = seq.calculate_kspace()
         k_traj_adc = torch.tensor(k_traj_adc_numpy, dtype=torch.float32)
 
-        n_samples = kheader.acq_info.number_of_samples
-        n_samples = torch.unique(n_samples)
-        if len(n_samples) > 1:
-            raise ValueError('We currently only support constant number of samples')
-        n_k0 = int(n_samples.item())
-
-        def reshape_pulseq_traj(k_traj: torch.Tensor, encoding_size: int):
-            k_traj *= encoding_size / (2 * torch.max(torch.abs(k_traj)))
+        def reshape(k_traj: torch.Tensor, encoding_size: int) -> torch.Tensor:
+            max_value_range = 2 * torch.max(torch.abs(k_traj))
+            if max_value_range > 1e-9 and encoding_size > 1:
```

Comment:
1e-9 seems very small for numerical errors of the trajectory in pulseq... @schuenke do we have any values from real-world examples?

Reply:
the 1e-9 is only to avoid overflow by division by zero (or almost zero), resulting in infs.

Reply:
made obsolete by #582